test/cypress/cypress/integration/step-definitions/management/rules/a-new-set-of-rules-displayed.when.js

```diff
@@ -4,7 +4,7 @@ import { RULES_PAGE as pageName } from '../../../utils/pages-constants';
 const paginatorSelector = getSelector('paginatorSelector', pageName);
 
 
-When('A new set of rules is displayed', () => {
+When('A new set of rules it\'s displayed', () => {
```

Comment:
This change is not necessary. `When a new set of rules is displayed` is correct.